crates/re_viewer_context/src/store_hub.rs

```diff
@@ -155,6 +155,19 @@ impl StoreHub {
         // If we have an app-id, then use it to look up the blueprint.
         let app_id = self.active_application_id.clone()?;
 
+        // Check that default and active blueprints exists,
+        // in case some of our book-keeping is broken.
```

Comment:
unclear to me when this would happen. Do we know why this is needed?

Reply:
It's not needed right now; it's defensive coding against future failures in book keeping.